esptfa_arima/Authentication/views.py

```
@@ -150,7 +150,7 @@ class StudentViewSet(ModelViewSet):
 
     queryset = Student.objects.all()
     serializer_class = StudentSerializer
-    permission_classes = [permissions.IsAuthenticated, IsAdminUser]
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)